src/Components/Scenes/ViewRequests/RequestTableBody.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import { Checkbox, TableCell, TableBody, TableRow } from '@material-ui/core';
+import colors from '@edma/design-tokens/js/color';
+import { makeStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import PropTypes from 'prop-types';
+
+const stableSort = (array, cmp) => {
```

Comment:
Do you think any of these sorting-related methods could be pulled out into a helpers file or something similar? That could potentially:
* Help with reusability if that ever comes up
* Greatly reduce the file length here
* Abstract away proven CompSci things we might not care about when glancing through component files

A little picky but could be nice, even if the functions are never re-used

Reply:
Great idea, I'm going to add these to a general purpose top-level utilities directory & file. I'm thinking maybe if we decide at a later time that we want to break it out into more specific files we can.